### ISSUE TITLE
Fisher doesn't load plugins

### ISSUE THREAD

**Issue by arwtyxouymz** [jorgebucaran/fisher#462]
Hi,
I use fisher and it's very fantastic plugin manager!

But one problem occur when i'm setting up new laptop.

### my problem
When i execute `fisher add .........` command and add plugins to fish shell, the plugins are recognized fish shell.
On the other hand, after close the terminal and open new one, fish doesn't recognize them.
why?
I tried login shell and interactive shell, but both are doesn't work.

`fisher ls` command returns my plugins.

```sh
~/.config/fisher/0rax/fish-bd
~/.config/fisher/edc/bass
~/.config/fisher/fisherman/getopts
~/.config/fisher/fisherman/spin
~/.config/fisher/fisherman/z
~/.config/fisher/oh-my-fish/plugin-peco
~/.config/fisher/oh-my-fish/theme-bobthefish
```

### my environment
- Ubuntu 16.04
- fish 2.7.1
- fisher 3.0.6

**Comment by jorgebucaran** [comment]
@arwtyxouymz What are the contents of your fishfile?

**Comment by arwtyxouymz** [comment]
@jorgebucaran 
I followed that guide.
And my fishfile is 
```fish
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

What is wrong?

**Comment by jorgebucaran** [comment]
@arwtyxouymz Your fishfile looks perfect. 

I don't know what's wrong. There is not enough information to make an assesment. I can see from the output of `fisher ls` that something is wrong.

What is the output of:

```
la ~/.config/fisher
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz Let's try something. Run`fisher` and paste the output logged to the terminal.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

Firstly, the output of `la ~/.config/fisher` is
```sh
合計 12K
drwxrwxr-x  3 ryutaro ryutaro 4.0K 10月  7 23:44 ./
drwx------ 24 ryutaro ryutaro 4.0K 10月  7 23:44 ../
drwxrwxr-x  6 ryutaro ryutaro 4.0K 10月  7 23:44 github.com/
```
sorry that output is japanese....


Next, the log of `fisher` command is
```sh
To completely erase z's data, remove:
/home/ryutaro/.local/share/z/datafetching https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master
fetching https://codeload.github.com/edc/bass/tar.gz/master
fetching https://codeload.github.com/0rax/fish-bd/tar.gz/master
fetching https://codeload.github.com/fisherman/spin/tar.gz/master
fetching https://codeload.github.com/fisherman/z/tar.gz/master
fetching https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master
linking __bobthefish_colors.fish
linking __bobthefish_display_colors.fish
linking __bobthefish_glyphs.fish
linking bobthefish_display_colors.fish
linking fish_greeting.fish
linking fish_mode_prompt.fish
linking fish_prompt.fish
linking fish_right_prompt.fish
linking fish_title.fish
linking __bass.py
linking bass.fish
linking bd.fish
linking uninstall.fish
linking bd.fish
linking spin.fish
linking spin.fish
linking __z.fish
linking __z_add.fish
linking __z_clean.fish
linking __z_complete.fish
linking uninstall.fish
linking z.fish
linking peco_kill.fish
linking peco_select_history.fish
fetching https://codeload.github.com/fisherman/getopts/tar.gz/master
linking getopts.fish
added 7 packages, removed 7 packages in 4682.35s
```

After `fisher` command, plugins are loaded.
But some `*.fish` files are generated to `$HOME` directory in spite of `$XDG_CONFIG_HOME` is set to `~/.config/`

**Comment by jorgebucaran** [comment]
@arwtyxouymz Hmm. Can you show me the value of the following variables?

```
echo $HOME
echo $XDG_CONFIG_HOME
echo $fisher_config
echo $fisher_cache
echo $fisher_path
```

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

`$HOME`
/home/ryutaro

`$XDG_CONFIG_HOME`
/home/ryutaro/.config/

`$fisher_config`

`$fisher_cache`

`$fisher_path`


`$fisher_*` is all empty, but i can use `fisher` command.

**Comment by jorgebucaran** [comment]
@arwtyxouymz The $fisher_* variables will be empty on a new shell, as they are not meant to be used by you. But if you run `fisher` they will become visible as they're global variables. Could you make sure they are really empty?

Start a new shell session and run `fisher` then show me the content of the variables again. They shouldn't be empty if everything is working normally.

**Comment by arwtyxouymz** [comment]
@jethrokuan 
After `fisher` command,

`$fisher_config`
/home/ryutaro/.config/fisher

`$fisher_cache`
/home/ryutaro/.cache/fisher

`$fisher_path`
/home/ryutaro/.config/fish

**Comment by jorgebucaran** [comment]
@arwtyxouymz Intriguing. Can I have your awk and sed versions? 

Also, does a tilde `~` expand to your home directory as usual?

```
echo ~
/home/ryutaro
```

**Comment by arwtyxouymz** [comment]
@jethrokuan 

`sed` version is `4.2.2`
`awk(mawk)` version is `1.3.3`

Yeah, tilde expand to home directory.

**Comment by jorgebucaran** [comment]
@arwtyxouymz This could be a problem with `IFS`! 🤔  

Can you go to [this line](https://github.com/jorgebucaran/fisher/blob/226c0d946fc87dc20c2eb49624815e6859ce7d9d/fisher.fish#L231):

https://github.com/jorgebucaran/fisher/blob/226c0d946fc87dc20c2eb49624815e6859ce7d9d/fisher.fish#L228-L234

...and add the following log:

```fish
echo ">> URL ($url)" >&2
echo ">> PKG ($pkg)" >&2
```

If my guess is right, `$url` should be correct, but `$pkg` will be empty on your system.

**Comment by jorgebucaran** [comment]
@arwtyxouymz Is this you [config.fish](https://github.com/arwtyxouymz/dotfiles/blob/04ee425e62735c34c305f5a3f3ed598233688ca9/.config/fish/config.fish#L2)? Only momentarily, remove [these lines](https://github.com/arwtyxouymz/dotfiles/blob/04ee425e62735c34c305f5a3f3ed598233688ca9/.config/fish/config.fish#L2-L22) and try running `fisher` again. Then paste the output here once again. 🙏

**Comment by jorgebucaran** [comment]
@arwtyxouymz Is this you [config.fish](https://github.com/arwtyxouymz/dotfiles/blob/04ee425e62735c34c305f5a3f3ed598233688ca9/.config/fish/config.fish#L2)? Only momentarily, remove [these lines](https://github.com/arwtyxouymz/dotfiles/blob/04ee425e62735c34c305f5a3f3ed598233688ca9/.config/fish/config.fish#L2-L22) and run `fisher` in a new shell session. Then shoe me the log again. 🙏

**Comment by arwtyxouymz** [comment]
@jethrokuan 
Is this function called when fish shell starts up?
When fish shell starts up, nothing is appeared.

But after `fisher` command, the output is this
```sh
To completely erase z's data, remove:
/home/ryutaro/.local/share/z/data>> URL (https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master)
>> PKG (github.com/oh-my-fish/theme-bobthefish)
>> URL (https://codeload.github.com/edc/bass/tar.gz/master)
>> PKG (github.com/edc/bass)
>> URL (https://codeload.github.com/0rax/fish-bd/tar.gz/master)
>> PKG (github.com/0rax/fish-bd)
>> URL (https://codeload.github.com/fisherman/spin/tar.gz/master)
>> PKG (github.com/fisherman/spin)
>> URL (https://codeload.github.com/fisherman/z/tar.gz/master)
>> PKG (github.com/fisherman/z)
fetching https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master
>> URL (https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master)
>> PKG (github.com/oh-my-fish/plugin-peco)
fetching https://codeload.github.com/edc/bass/tar.gz/master
fetching https://codeload.github.com/0rax/fish-bd/tar.gz/master
fetching https://codeload.github.com/fisherman/spin/tar.gz/master
fetching https://codeload.github.com/fisherman/z/tar.gz/master
fetching https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master
linking __bobthefish_colors.fish
linking __bobthefish_display_colors.fish
linking __bobthefish_glyphs.fish
linking bobthefish_display_colors.fish
linking fish_greeting.fish
linking fish_mode_prompt.fish
linking fish_prompt.fish
linking fish_right_prompt.fish
linking fish_title.fish
linking __bass.py
linking bass.fish
linking bd.fish
linking uninstall.fish
linking bd.fish
linking spin.fish
linking spin.fish
linking __z.fish
linking __z_add.fish
linking __z_clean.fish
linking __z_complete.fish
linking uninstall.fish
linking z.fish
linking peco_kill.fish
linking peco_select_history.fish
>> URL (https://codeload.github.com/fisherman/getopts/tar.gz/master)
>> PKG (github.com/fisherman/getopts)
fetching https://codeload.github.com/fisherman/getopts/tar.gz/master
linking getopts.fish
added 7 packages, removed 7 packages in -2821.75s
```

**Comment by arwtyxouymz** [comment]
@jethrokuan 

Yeah, but it's my OS X version `config.fish`.
Now i use Ubuntu 16.04, and my `config.fish` is 
```fish
set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
for file in $XDG_CONFIG_HOME/fish/conf.d/*.fish
    buildin source $file 2>/dev/null
end

# tmux
if status is-interactive; and not set -q TMUX
    set create_new_session "new"
    if tmux ls > /dev/null ^ /dev/null
        set IFS
        set ID (tmux ls)
        set IFS \n" "\t
        set ID "$ID\n$create_new_session:"
    else
        set ID "$create_new_session:"
    end

    set ID (echo -e $ID | peco | cut -d: -f1)

    if test "$ID" = "$create_new_session"
        tmux new-session
    else if test -n "$ID"
        tmux attach -t $ID
    else
        echo "Start as a normal shell"
    end
end

if not functions -q fisher
    echo "Installing fisher for the first time..." >&2
    set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
    curl https://git.io/fisher --create-dirs -sLo $XDG_CONFIG_HOME/fish/functions/fisher.fish
    fisher
end

# キーバインディング
function fish_user_key_bindings
  bind \cr peco_select_history
end

# cd のあとに ls -a
function cd
  builtin cd $argv
  ls -a
end

# theme設定
set -g theme_display_git yes
set -g theme_display_git_untracked yes
set -g theme_display_git_master_branch yes
set -g theme_title_use_abbreviated_path no
set -g fish_prompt_pwd_dir_length 0
set -g theme_project_dir_length 0
set -g theme_newline_cursor yes

alias vi nvim
alias vim nvim

# cmakeプロジェクトの初期化
function CppInitialize
    mkdir -p ./build/
    touch ./CMakeLists.txt
    touch ./main.cpp
end

# build
function build
    if test -x (command -v clang)
        cmake .. -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
        if test -x (command -v compdb)
            compdb list > ../compile_commands.json
        else
            echo "\e[33mWarning: compdb is not installed"
        end
    else
        cmake ..
        echo "\e[33mWarning: clang is not installed, using default compiler"
    end
end
```

And after i delete about tmux line that you specified, the `fisher` output is 
```sh
To completely erase z's data, remove:
/home/ryutaro/.local/share/z/data>> URL (https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master)
>> PKG (github.com/oh-my-fish/theme-bobthefish)
>> URL (https://codeload.github.com/edc/bass/tar.gz/master)
>> PKG (github.com/edc/bass)
>> URL (https://codeload.github.com/0rax/fish-bd/tar.gz/master)
>> PKG (github.com/0rax/fish-bd)
>> URL (https://codeload.github.com/fisherman/spin/tar.gz/master)
>> PKG (github.com/fisherman/spin)
>> URL (https://codeload.github.com/fisherman/z/tar.gz/master)
>> PKG (github.com/fisherman/z)
fetching https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master
>> URL (https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master)
fetching https://codeload.github.com/edc/bass/tar.gz/master
>> PKG (github.com/oh-my-fish/plugin-peco)
fetching https://codeload.github.com/0rax/fish-bd/tar.gz/master
fetching https://codeload.github.com/fisherman/spin/tar.gz/master
fetching https://codeload.github.com/fisherman/z/tar.gz/master
fetching https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master
linking __bobthefish_colors.fish
linking __bobthefish_display_colors.fish
linking __bobthefish_glyphs.fish
linking bobthefish_display_colors.fish
linking fish_greeting.fish
linking fish_mode_prompt.fish
linking fish_prompt.fish
linking fish_right_prompt.fish
linking fish_title.fish
linking __bass.py
linking bass.fish
linking bd.fish
linking uninstall.fish
linking bd.fish
linking spin.fish
linking spin.fish
linking __z.fish
linking __z_add.fish
linking __z_clean.fish
linking __z_complete.fish
linking uninstall.fish
linking z.fish
linking peco_kill.fish
linking peco_select_history.fish
>> URL (https://codeload.github.com/fisherman/getopts/tar.gz/master)
>> PKG (github.com/fisherman/getopts)
fetching https://codeload.github.com/fisherman/getopts/tar.gz/master
linking getopts.fish
added 7 packages, removed 7 packages in -2545.74s
```

**Comment by jorgebucaran** [comment]
> @arwtyxouymz Is this function called when fish shell starts up?

No, it is not. Nothing is run on shell startup. Please read my comment above.

And by the way, my GitHub username is:

### → @jorgebucaran

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

I'm so sorry........
I misunderstood ....

**Comment by jorgebucaran** [comment]
@arwtyxouymz It looks like `$IFS` was not the problem after all. Good to know.

The problem is somewhere in this function:

https://github.com/jorgebucaran/fisher/blob/226c0d946fc87dc20c2eb49624815e6859ce7d9d/fisher.fish#L292-L319

**Comment by jorgebucaran** [comment]
@arwtyxouymz Fisher is downloading packages to the correct location, but there is obviously a problem when linking. Also, your `fisher ls` output seems wrong.

Can you confirm that `fisher ls` produces the following output?
```console
~/.config/fisher/0rax/fish-bd
~/.config/fisher/edc/bass
~/.config/fisher/fisherman/getopts
~/.config/fisher/fisherman/spin
~/.config/fisher/fisherman/z
~/.config/fisher/oh-my-fish/plugin-peco
~/.config/fisher/oh-my-fish/theme-bobthefish
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz I'm starting to think this could be a problem with `sed`.

https://github.com/jorgebucaran/fisher/blob/226c0d946fc87dc20c2eb49624815e6859ce7d9d/fisher.fish#L82-L87

**Comment by jorgebucaran** [comment]
@arwtyxouymz I'm starting to think this could be a problem with `sed`.

Please run this and show me the output.

```fish
printf "%s\n" $fisher_config/*/*/* | sed "s|$fisher_config/||;s|github\.com/||" 
```

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

Yeah `fisher ls` output is that one.

```
printf "%s\n" $fisher_config/*/*/* | sed "s|$fisher_config/||;s|github\.com/||" 
```
output is 
```
/home/ryutaro/.config/fisher/0rax/fish-bd
/home/ryutaro/.config/fisher/edc/bass
/home/ryutaro/.config/fisher/fisherman/getopts
/home/ryutaro/.config/fisher/fisherman/spin
/home/ryutaro/.config/fisher/fisherman/z
/home/ryutaro/.config/fisher/oh-my-fish/plugin-peco
/home/ryutaro/.config/fisher/oh-my-fish/theme-bobthefish
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz Okay, and what about:

```fish
printf "%s\n" $fisher_config/*/*/*
```

**Comment by arwtyxouymz** [comment]
@jorgebucaran 
It's same with just before command.
```sh
/home/ryutaro/.config/fisher/github.com/0rax/fish-bd
/home/ryutaro/.config/fisher/github.com/edc/bass
/home/ryutaro/.config/fisher/github.com/fisherman/getopts
/home/ryutaro/.config/fisher/github.com/fisherman/spin
/home/ryutaro/.config/fisher/github.com/fisherman/z
/home/ryutaro/.config/fisher/github.com/oh-my-fish/plugin-peco
/home/ryutaro/.config/fisher/github.com/oh-my-fish/theme-bobthefish
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz The sed command is not producing the right output. Only `github.com` is removed from the string. But $fisher_config (`/home/ryutaro/.config/fisher`) is not.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

First one is 
```sh
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

Second is
```
/home/ryutaro/.config/fisher/0rax/fish-bd
/home/ryutaro/.config/fisher/edc/bass
/home/ryutaro/.config/fisher/fisherman/getopts
/home/ryutaro/.config/fisher/fisherman/spin
/home/ryutaro/.config/fisher/fisherman/z
/home/ryutaro/.config/fisher/oh-my-fish/plugin-peco
/home/ryutaro/.config/fisher/oh-my-fish/theme-bobthefish
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz They should have the same output. Isn't `$fisher_config` → `/home/ryutaro/.config/fisher`?

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

Yes.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

It's sed problem?
Should i reinstall sed?

**Comment by jorgebucaran** [comment]
Based on this https://github.com/jorgebucaran/fisher/issues/462#issuecomment-427666462 it doesn't look like a `sed` problem. おもしろいねどうしよう 😆

**Comment by jorgebucaran** [comment]
@arwtyxouymz Let's try something simpler:

```fish
echo $fisher_config | sed "s|$fisher_config||"
```

That should print an empty string.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 
Exactly an empty string.

**Comment by jorgebucaran** [comment]
Please confirm also:

```fish
test $fisher_config = /home/ryutaro/.config/fisher; and echo OK
```

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

Yeah it's OK

**Comment by jorgebucaran** [comment]
Could it be a problem with multiple lines? What if we echo only one line?

```fish
echo /home/ryutaro/.config/fisher/github.com/edc/bass | sed "s|$fisher_config/||"
```

Should print

```console
github.com/edc/bass
```

**Comment by arwtyxouymz** [comment]
This command prints
```sh
github.com/edc/bass
```
correctly.

**Comment by jorgebucaran** [comment]
One line works. What about multiple lines?

```fish
printf "%s\n" /home/ryutaro/.config/fisher/github.com/edc/bass /home/ryutaro/.config/fisher/fisherman/z | sed "s|$fisher_config/||"
```

**Comment by arwtyxouymz** [comment]
```sh
github.com/edc/bass
fisherman/z
```

is output.

**Comment by jorgebucaran** [comment]
That's what I expect. Let's see.

Printf a glob expansion.

```fish
printf "%s\n" $fisher_config/*/*/*
```

Then pipe it to sed.

```fish
printf "%s\n" $fisher_config/*/*/* | sed "s|$fisher_config/||" 
```

Then pipe it to sed using multiple replacements.
```fish
printf "%s\n" $fisher_config/*/*/* | sed "s|$fisher_config/||;s|github\.com/||" 
```

**Comment by arwtyxouymz** [comment]
glob expansion:
```sh
/home/ryutaro/.config/fisher/github.com/0rax/fish-bd
/home/ryutaro/.config/fisher/github.com/edc/bass
/home/ryutaro/.config/fisher/github.com/fisherman/getopts
/home/ryutaro/.config/fisher/github.com/fisherman/spin
/home/ryutaro/.config/fisher/github.com/fisherman/z
/home/ryutaro/.config/fisher/github.com/oh-my-fish/plugin-peco
/home/ryutaro/.config/fisher/github.com/oh-my-fish/theme-bobthefish
```

sed:
```sh
github.com/0rax/fish-bd
github.com/edc/bass
github.com/fisherman/getopts
github.com/fisherman/spin
github.com/fisherman/z
github.com/oh-my-fish/plugin-peco
github.com/oh-my-fish/theme-bobthefish
```

multiple replacements:
```sh
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

**Comment by jorgebucaran** [comment]
@arwtyxouymz That's not what you said here in this https://github.com/jorgebucaran/fisher/issues/462#issuecomment-427666462.

**Comment by jorgebucaran** [comment]
@arwtyxouymz You are giving me contradictory information.

Let's try again. But first to be sure:

```fish
type _fisher_ls
```

Okay, now, please run and show me your output.

```fish
printf "%s\n" $fisher_config/*/*/* | sed "s|$fisher_config/||;s|github\.com/||" 
```

```fish
for pkg in $fisher_config/*/*/*
    command readlink $pkg
    and continue
    or echo $pkg
end | command sed "s|$fisher_config/||;s|github\.com/||"
```

```fish
_fisher_ls
```

```fish
fisher ls
```

They all should have the same output.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

I think that's because i changed `XDG_CONFIG_HOME` from `/home/ryutaro/.config/` to `/home/ryutaro/.config`.

I didn't think this change cause big difference in sed command.

**Comment by jorgebucaran** [comment]
@arwtyxouymz Ah, okay that explains the problem. I think I can fix that by making the trailing slash `/` optional in the sed regex.

**Comment by jorgebucaran** [comment]
@arwtyxouymz Is everything working now?

**Comment by arwtyxouymz** [comment]
First one is 
```sh
# Defined in /home/ryutaro/.config/fish/functions/fisher.fish @ line 82
function _fisher_ls
    set -l pkgs $fisher_config/*/*/*
    for pkg in $pkgs
        command readlink $pkg
        and continue
        or echo $pkg
    end | command sed "s|$fisher_config/||;s|github\.com/||"
end
```
Second one (`printf` one) is
```sh
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

Third one (`for` statement) is
```
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

Forth one (`_fisher_ls`) is
```
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

Fifth one (`fisher ls`) is
```
0rax/fish-bd
edc/bass
fisherman/getopts
fisherman/spin
fisherman/z
oh-my-fish/plugin-peco
oh-my-fish/theme-bobthefish
```

**Comment by arwtyxouymz** [comment]
And plugins are loaded!

Thank you so much!!!

What was a problem?

**Comment by jorgebucaran** [comment]
@arwtyxouymz Just to be double sure. Could you run `fisher` and show me the log one more time now that everything seems to be working correctly?

**Comment by jorgebucaran** [comment]
> @arwtyxouymz What was a problem?

Problem was fixed by you changing XDG_CONFIG_HOME from `/home/ryutaro/.config/` to `/home/ryutaro/.config`. So, the problem was the extra `/` at the end of the path. I can fix that by making the slash optional in the regex.

**Comment by arwtyxouymz** [comment]
@jorgebucaran 

`fisher` output is
```
To completely erase z's data, remove:
/home/ryutaro/.local/share/z/data>> URL (https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master)
>> PKG (github.com/oh-my-fish/theme-bobthefish)
>> URL (https://codeload.github.com/edc/bass/tar.gz/master)
>> PKG (github.com/edc/bass)
>> URL (https://codeload.github.com/0rax/fish-bd/tar.gz/master)
>> PKG (github.com/0rax/fish-bd)
>> URL (https://codeload.github.com/fisherman/spin/tar.gz/master)
>> PKG (github.com/fisherman/spin)
fetching https://codeload.github.com/oh-my-fish/theme-bobthefish/tar.gz/master
>> URL (https://codeload.github.com/fisherman/z/tar.gz/master)
>> PKG (github.com/fisherman/z)
>> URL (https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master)
>> PKG (github.com/oh-my-fish/plugin-peco)
fetching https://codeload.github.com/edc/bass/tar.gz/master
fetching https://codeload.github.com/0rax/fish-bd/tar.gz/master
fetching https://codeload.github.com/fisherman/spin/tar.gz/master
fetching https://codeload.github.com/fisherman/z/tar.gz/master
fetching https://codeload.github.com/oh-my-fish/plugin-peco/tar.gz/master
linking ~/.config/fish/functions/__bobthefish_colors.fish
linking ~/.config/fish/functions/__bobthefish_display_colors.fish
linking ~/.config/fish/functions/__bobthefish_glyphs.fish
linking ~/.config/fish/functions/bobthefish_display_colors.fish
linking ~/.config/fish/functions/fish_greeting.fish
linking ~/.config/fish/functions/fish_mode_prompt.fish
linking ~/.config/fish/functions/fish_prompt.fish
linking ~/.config/fish/functions/fish_right_prompt.fish
linking ~/.config/fish/functions/fish_title.fish
linking ~/.config/fish/functions/__bass.py
linking ~/.config/fish/functions/bass.fish
linking ~/.config/fish/functions/bd.fish
linking ~/.config/fish/completions/bd.fish
linking ~/.config/fish/completions/spin.fish
linking ~/.config/fish/functions/spin.fish
linking ~/.config/fish/functions/__z.fish
linking ~/.config/fish/functions/__z_add.fish
linking ~/.config/fish/functions/__z_clean.fish
linking ~/.config/fish/functions/__z_complete.fish
linking ~/.config/fish/conf.d/z.fish
linking ~/.config/fish/functions/peco_kill.fish
linking ~/.config/fish/functions/peco_select_history.fish
>> URL (https://codeload.github.com/fisherman/getopts/tar.gz/master)
>> PKG (github.com/fisherman/getopts)
fetching https://codeload.github.com/fisherman/getopts/tar.gz/master
linking ~/.config/fish/functions/getopts.fish
updated 7 packages in 2146.16s
```

Oh Thank you!
I'm really glad to use useful plugins.

**Comment by jorgebucaran** [comment]
@arwtyxouymz Glad to hear that! 

BTW I noticed there's an issue with the printed time:

```
updated 7 packages in 2146.16s
```

I'll fix that too later. 

I'll mention your name here when I have released a new version so you can self-update.

Thank you for your patience!

**Comment by jorgebucaran** [comment]
@arwtyxouymz Fixed trailing slash error in 3.0.7 2549ac1. 🎉 

Update to the latest fisher using:

```
fisher self-update
```

Cheers!
